cypress/e2e/tests/migration/applicationinventory/analysis/custom_metrics_after_bulk_analysis.test.ts

```diff
@@ -47,11 +51,20 @@ describe(["@tier4"], "Bulk Analysis", () => {
                 analyses.forEach((analysis) => analysis.create());
             });
         });
+
+        // Get the current counter value
+        metrics.getValue(metricName).then((counterValue) => {
+            counter = counterValue;
+        });
     });
 
-    it("Bulk analysis of source code + open source deps + known deps", function () {
+    it("Count the tasks after bulk analysis of source code + open source deps + known deps", function () {
         Analysis.analyzeAll(analyses[0]);
         Analysis.verifyAllAnalysisStatuses(AnalysisStatuses.completed);
+        counter = counter + 25;
```

Comment:
use `counter + NUMBER_OF_APPS` so the test won't fail if we change the number of applications analyzed

Reply:
Thanks for noticing this, Done